navigation/MainNavigator.js

```diff
@@ -30,9 +29,9 @@ function MainNavigator() {
             case "Profile":
               iconName = "ios-contact";
               break;
-            case "Playground":
-              iconName = "md-happy";
-              break;
+            // case "Playground":
```

Comment:
Still want to keep it? or delete all playground?

Reply:
It served it's purpose. I'm okay with deleting all code related to playground in another PR.